_posts/2021-01-13-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,34 @@
+author: flavius
+categories: [ audio ]
+image: ""
+featured: false
+---
+What is audio mixing? Is it helpful or a hindrance? What are my options for mixing audio?
+How much does it cost?
+
+### - Introduction
+
+Audio mixing, what is it? Audio Mixing is where you take multiple input sources of audio and put them into a single or fewer number output signals. An example of input sources would be a microphone, game audio, stream alert audio. An example of an output signal would be our headphones, speakers, or stream. Can you achieve audio mixing for streams and videos without a digital or physical audio mixer? Mostly, but in practice it is significantly more complicated to route audio properly without using some form of audio mixing software or hardware when dealing with multiple sources.
+
+### - Goals of Audio Mixing
+
+The goals of audio mixing can be as simple as taking a microphone and game audio then pushing it to a single output. They can also be as elaborate as dividing out a microphone, or even two or three microphones, game audio, alerts, computer sounds, and music. Each having their own source to pull audio from then pushing that audio to two or more outputs, headphones, broadcast software such as OBS or SLOBS, or discord. Of course we don't want our stream audio going to friends on discord, and we don't want our mic audio going to our speakers if we aren't using headphones at the moment.
+
+This is where audio mixing software and hardware comes into play. Audio mixing allows us to properly route our audio to where it needs to be and often times gives it a boost in quality, while also making it easier to handle unlike windows sound settings and such. Having more options can be daunting at times, especially when just starting out, but once you get the hang of it, more often than not it saves so much more time and provides you with better quality than if you just went without.
+
+### - Audio Mixing options
+
+Are you interested in having more control over your audio? Lets go over our options then! I'm sure every streamer has heard of a GoXLR, or GoXLR-Mini which come with software mixing alongside hardware controls. There is also the Elgato Wave microphones which come with similar software mixing and some hardware controls. Physical mixers are also available of various sizes and prices that stretch from less than $50 US Dollars to well over $500 to even $1000s. These beastly devices offer physical nobs and sliders for customization and take up a bit more space than a GoXLR but in some cases not by much or at all as some of them can be fairly small. However, their prices range from significantly lower than even a GoXLR-Mini or even cheaper then the Elgato Wave. Then finally, there is Voicemeeter, something either mostly unheard of or often not spoken. What is Voicemeeter? It is a digital mixing software for windows designed to help thread and control audio similar to a physical mixer, except that it is an application ran on windows.
```

Comment:
Insert a paragraph break before "Physical mixers are also available...".

Reply:
Incomplete sentence at "However, their prices range from significantly lower...".

Reply:
I'd insert another paragraph break at "Then finally, there is Voicemeeter,". I'd also drop the "Then" from the start of that sentence. This will make it feel like a "breath of fresh air".

Reply:
Rather than having the question then immediately answering it (for example, "What is Voicemeeter? It is a digital mixing..."), drop the question and jump straight into the answer. It's kind of like narrating someone else's monologue when the question is explicitly written out. 

If you want to include the question, you could turn it into a heading instead. This would break up the article a bit more and give people "anchor points" so they can "mentally" take a break. It also can be less intimidating.

Reply:
this is different in the rewritten separated articles and needs a new review on it.